setup.py

```diff
@@ -53,6 +53,6 @@
         '': ['*.eml'],
         },
     test_suite='metaswitch.ellis.test',
-    install_requires=["py-bcrypt", "tornado==2.3", "msgpack-python", "phonenumbers", "SQLAlchemy", "MySQL-python"],
+    install_requires=["py-bcrypt", "tornado==2.3", "msgpack-python", "phonenumbers", "SQLAlchemy", "MySQL-python", "prctl"],
```

Comment:
Yep - the packaging scripts run setup.py to install the egg dependencies locally, then package them into the .deb file. Our postint script only use the eggs installed by the .deb file - they're explicitly forbidden to go to PyPI.